pkg/kubectl/cmd/delete.go

```diff
@@ -218,6 +218,10 @@ func (o *DeleteOptions) Validate(cmd *cobra.Command) error {
 			o.WaitForDeletion = true
 			o.GracePeriod = 1
 		}
+	} else {
+		if o.ForceDeletion {
+			return fmt.Errorf("--force should be specified with --grace-period=0")
```

Comment:
I'd prefer to print warnings instead of errors.

Reply:
@dixudx Done.